src/components/shared/button/index.tsx

```diff
@@ -7,15 +7,17 @@ interface Props {
   handleSubmit?: (event: GestureResponderEvent) => void;
   style?: StyleProp<ViewStyle>;
   isLoading?: boolean;
+  isDisabled?: boolean;
 }
 
-const Button = ({text, handleSubmit, style, isLoading}: Props) => {
+const Button = ({text, handleSubmit, style, isLoading, isDisabled}: Props) => {
   return (
     <ButtonNativeBase
       onPress={handleSubmit}
       w={'50%'}
       style={style}
-      isLoading={isLoading}>
+      isLoading={isLoading}
+      disabled={isDisabled}>
```

Comment:
here we should use the `isDisabled` prop

Reply:
Done.